include/networkit/centrality/DynApproxBetweenness.hpp

```diff
@@ -25,61 +24,64 @@ namespace NetworKit {
  * @ingroup centrality
  * Interface for dynamic approximated betweenness centrality algorithm.
  */
-class DynApproxBetweenness: public Centrality, public DynAlgorithm {
+class DynApproxBetweenness : public Centrality, public DynAlgorithm {
 
 public:
     /**
-      * The algorithm approximates the betweenness of all vertices so that the scores are
-      * within an additive error @a epsilon with probability at least (1- @a delta).
-      * The values are normalized by default.
-      *
-      * @param	G			the graph
-      * @param	storePredecessors   keep track of the lists of predecessors?
-      * @param	epsilon		maximum additive error
-      * @param	delta		probability that the values are within the error guarantee
-      * @param	universalConstant	the universal constant to be used in
-      * computing the sample size. It is 1 by default. Some references suggest
-      * using 0.5, but there is no guarantee in this case.
+     * The algorithm approximates the betweenness of all vertices so that the scores are
+     * within an additive error @a epsilon with probability at least (1- @a delta).
+     * The values are normalized by default.
+     *
+     * @param	G			the graph
+     * @param  storePredecessors   keep track of the lists of predecessors?
+     * @param	epsilon		maximum additive error
+     * @param	delta		probability that the values are within the error guarantee
      */
     DynApproxBetweenness(const Graph &G, double epsilon = 0.01, double delta = 0.1,
-                         bool storePredecessors = true, double universalConstant = 1.0);
+                         bool storePredecessors = true, double universalConstant = 0.5);
 
     /**
      * Runs the static approximated betweenness centrality algorithm on the initial graph.
      */
     void run() override;
 
     /**
-    * Updates the betweenness centralities after an edge insertions on the graph.
-    * Notice: it works only with edge insertions and the graph has to be connected.
-    *
-    * @param e The edge insertions.
-    */
+     * Updates the betweenness centralities after an edge insertions on the graph.
+     * Notice: it works only with edge insertions and the graph has to be connected.
+     *
+     * @param e The edge insertions.
```

Comment:
The fact that this algorithm supports insertions only should be mentioned in the documentation above the constructor, I think. Also, since this function takes a single edge update, it should be:
```suggestion
     * @param e The edge insertion.
```

Reply:
Actually, the algorithm can handle deletions now too that the DynSSSP algorithms can handle deletions. Updated the documentation accordingly.